qpc/scan/job.py

```diff
@@ -42,7 +41,7 @@ def __init__(self, subparsers):
                             scan.SCAN_URI, [codes.ok])
         self.parser.add_argument('--name', dest='name', metavar='NAME',
                                  help=_(messages.SCAN_NAME_HELP),
-                                 required=True)
+                                 required=False)
```

Comment:
Should be mutually exclusive:
https://github.com/quipucords/quipucords/blob/master/qpc/cred/add.py#L56